front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/InputsTable.test.tsx

```diff
@@ -163,6 +167,14 @@ describe('Insights Input Table', () => {
       render(<InputsTable />);
       expect(screen.getAllByTestId('insightsInputsTableRow')).toHaveLength(2);
     });
+    it('adds previewedInputId to url correctly on row click', () => {
```

Comment:
Would have been good to add some tests on moveUp and moveDown but I understand time is limited, the logic is indeed concise in the end, but not so easily understood for future devs coming back to this, tests could help discover and maintain the mechanism.

Reply:
Indeed! Completely agree with you on that but also, we hit the limit of unit testing a bit with this functionality. This is partly why I mentioned e2e tests for this flow earlier, this behaviour is much better tested with Cypress, imo. Let's do out best to find the time to cypress test it in the future :)